### PR TITLE
Add a conformance test for MultipleInputFeatureRequirement on Workflow outputs

### PR DESCRIPTION
**Pull request by kinow**
For https://github.com/common-workflow-language/common-workflow-language/issues/637

Tried to write a test that calls `echo-tool.cwl` twice, storing the output, and then uses both values in the workflow outputs. Confirmed it fails without the `MultipleInputFeatureRequirement`.

### CODE REVIEW

File: conformance_tests.yaml [hunk 1]
```diff
@@ -3532,3 +3532,15 @@
   doc: |
     Use of $(runtime.outdir) for outputBinding glob.
   tags: [ required, command_line_tool ]
+
+- label: multiple-input-feature-requirement
```
Comment (kinow) [inline conformance_tests.yaml:3536]: p.s.: was reading the doc for conformance tests, and noticed this line:

>label: a short list of underscore (_) separated words that succinctly identifies and explains the test.

I had already used dashes, as I did in the previous test, where I followed the pattern of the previous test… Is there a reason for underscores? Should we fix the existing labels? Or maybe update https://github.com/common-workflow-language/cwl-v1.2/blob/1.2.1_proposed/CONFORMANCE_TESTS.md?

Comment (mr-c) [reply conformance_tests.yaml:3536]: Hmm.. I don't have a strong opinion right now, except that we should be consistent. Can you make a separate issue to discuss this?

**Review by mr-c — APPROVED** [review]

File: conformance_tests.yaml (hunk 1 above)
Comment (kinow) [reply conformance_tests.yaml:3536]: Agree! Done: https://github.com/common-workflow-language/cwl-v1.2/issues/191

Thanks

**Comment by kinow** [comment]
Rebased and fixed conflicts.

**Pull request merged**